src/Microsoft.DocAsCode.MarkdigEngine.Extensions/CodeSnippet/HtmlCodeSnippetRenderer.cs

```diff
@@ -176,6 +176,8 @@ private void AddExtractorItem(string language, CodeSnippetExtrator extractor)
         protected override void Write(HtmlRenderer renderer, CodeSnippet codeSnippet)
         {
             var (content, codeSnippetPath) = _context.ReadFile(codeSnippet.CodePath, InclusionContext.File, codeSnippet);
+            
+            _context.LogWarning("code-snippet-deprecated", "The legacy code snippet inclusion will be deprecated at a later date. Please use the new code extension. (http://aka.ms/codesnippet)", codeSnippet);
```

Comment:
Based on the comments in the linked work item, I see this change should not block completing any build currently. Some repos like `azure-docs-pr` blocks a pull request if it contains a warning. Should this be `_context.LogSuggestion`? We can change it back to `_context.LogWarning` when it is ready to be enforeced.

Reply:
Sounds good @superyyrrzz

Reply:
Done @superyyrrzz !